### PR TITLE
Refine component styles

### PR DESCRIPTION
**Pull request by corwinharrell**

### CODE REVIEW

File: src/components/Avatar/Avatar.module.scss [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 .avatar {
   display: flex;
   align-items: center;
 }
 
 .textWrapper {
-  margin-left: 5px;
-}
\ No newline at end of file
+  margin-left: $m-spacing;
```
Comment (villanuevawill) [inline src/components/Avatar/Avatar.module.scss]: I loooveee the switch to spacing variables.

File: src/components/Button/index.js [hunk 2]
```diff
@@ -58,11 +58,11 @@ Button.propTypes = {
   className: PropTypes.string,
   type: PropTypes.oneOf([
     'primary',
-    'secondary',
+    'default',
     'destructive',
     'action',
     'link',
-    'delete'
+    'link-destructive'
```
Comment (villanuevawill) [inline src/components/Button/index.js]: I like the update to link-destructive.

File: src/components/Circle/index.js [hunk 3]
```diff
@@ -60,16 +64,8 @@ const Circle = props => {
...
-    let blockyScale = 4;
-    if (size === 'small') {
-      blockySize = 13;
-      blockyScale = 4;
-    }
-    if (size === 'large') {
-      blockySize = 40;
-      blockyScale = 5;
-    }
+    let blockySize = 15;
```
Comment (villanuevawill) [inline src/components/Circle/index.js]: Awesome - keeping the blockyscale fixed is better as now we actually have a consistent blockie.

File: src/components/FileUpload/FileUpload.module.scss [hunk 4]
```diff
@@ -38,6 +49,7 @@
 }
 
 .sizeText {
+  color: $brand-grey;
   margin: 0 5px;
```
Comment (villanuevawill) [inline src/components/FileUpload/FileUpload.module.scss]: @corwinharrell do we need this margin to be a variable?

File: src/components/ListGroup/ListGroup.module.scss [hunk 5]
```diff
@@ -1,15 +1,29 @@
 @import "~styles/variables.scss";
 
 .listItem {
-  padding: 10px 0;
-  border-bottom: 1px solid $base-border-color;
+  padding: $m-spacing $m-spacing;
+  position: relative;
+  border-radius: $base-border-radius;
+  margin-top: -1px;
```
Comment (villanuevawill) [inline src/components/ListGroup/ListGroup.module.scss]: Same here - do we want to use spacing variables?

File: src/components/NumberInput/NumberInput.module.scss [hunk 6]
```diff
@@ -4,59 +4,70 @@
...
-  width: 80px;
-  padding: 0 12px;
+  padding: 0 $base-spacing;
   vertical-align: top;
   text-align: center;
   outline: none;
-  background: $brand-lightGrey;
+  background: $base-input-background;
+  border: $base-border;
+  border-width: 1px 0;
```
Comment (villanuevawill) [inline src/components/NumberInput/NumberInput.module.scss]: border width var?

File: src/components/RadioGroup/RadioGroup.module.scss [hunk 7]
```diff
@@ -59,10 +59,10 @@
 
   /* Style the indicator (dot/circle) */
   .customRadio:after {
-    top: $base-checkbox-size / 5.33;
-    left: $base-checkbox-size / 5.33;
-    width: $base-checkbox-size * 0.625;
-    height: $base-checkbox-size * 0.625;
+    top: $base-checkbox-size * 0.15;
```
Comment (villanuevawill) [inline src/components/RadioGroup/RadioGroup.module.scss]: fascinating

File: src/components/Tabs/Tabs.module.scss [hunk 8]
```diff
@@ -4,43 +4,41 @@
...
   display: flex;
   flex-direction: row;
-  justify-content: space-around;
   border-bottom: 1px solid $brand-lightGrey;
 }
 
 .tab {
-  flex-grow: 1;
-  text-align: center;
+  display: flex;
```
Comment (villanuevawill) [inline src/components/Tabs/Tabs.module.scss]: so for the justify option - we would just set this back to flex-grow. Cool

File: src/components/Text/index.js [hunk 9]
```diff
@@ -3,58 +3,73 @@ import PropTypes from 'prop-types';
...
-      </span>
+        {props.children}
+      </a>
+    );
+  }
+
+  if (inputLabel) {
+    addedClasses += `${styles.inputLabel}`;
+
+    return (
```
Comment (villanuevawill) [inline src/components/Text/index.js]: @corwinharrell why not make inputLabel a typeScale?

File: src/stories/Button.stories.js [hunk 10]
```diff
@@ -6,175 +6,178 @@ import { action } from '@storybook/addon-actions';
...
+      className={'sb-component-group-description'}
+      typeScale="Body"
+      lineHeight="lineHeight-default"
+    >
+      The <code>fitWidth</code> prop will cause the button to span teh
+      full-width of its parent container.
+    </Text>
+
+    <div class="sb-component-group">
+      <Button fitWidth type="primary">
```
Comment (villanuevawill) [inline src/stories/Button.stories.js]: amazing.

File: src/stories/Text.stories.js [hunk 11]
```diff
@@ -1,97 +1,241 @@
...
+        <Text typeScale="Body" style="italic">
+          Italic
+        </Text>
+      </div>
+
+      <div class="sb-component-container">
+        <Text typeScale="Body" style="uppercase">
+          Uppercase
+        </Text>
+      </div>
```
Comment (villanuevawill) [inline src/stories/Text.stories.js]: Also amazing... good work here

File: src/styles/Toggle.scss [hunk 12]
```diff
@@ -38,104 +40,60 @@
...
+.react-toggle--focus .react-toggle-track {
+  box-shadow: 0 0 0 2px $brand-blue;
 }
 
-.react-toggle:active:not(.react-toggle--disabled) .react-toggle-thumb {
-  -webkit-box-shadow: 0px 0px 5px 5px #0099E0;
-  -moz-box-shadow: 0px 0px 5px 5px #0099E0;
-  box-shadow: 0px 0px 5px 5px #0099E0;
+.react-toggle:active:not(.react-toggle--disabled) .react-toggle-track {
+  box-shadow: 0 0 0 2px $brand-blue;
```
Comment (villanuevawill) [inline src/styles/Toggle.scss]: this looks beautiful.

File: src/components/FileUpload/FileUpload.module.scss (hunk 4 above)
Comment (corwinharrell) [reply src/components/FileUpload/FileUpload.module.scss]: Yeah we can change this.

File: src/components/ListGroup/ListGroup.module.scss (hunk 5 above)
Comment (corwinharrell) [reply src/components/ListGroup/ListGroup.module.scss]: Yep, I recently created a $base-border-width variable so will go through and replace instances like this 👍

File: src/components/NumberInput/NumberInput.module.scss (hunk 6 above)
Comment (corwinharrell) [reply src/components/NumberInput/NumberInput.module.scss]: 👍

File: src/components/Tabs/Tabs.module.scss (hunk 8 above)
Comment (corwinharrell) [reply src/components/Tabs/Tabs.module.scss]: yessir

File: src/components/Text/index.js (hunk 9 above)
Comment (corwinharrell) [reply src/components/Text/index.js]: The goal here is really just to restrict how inputLabels appear. It ultimately uses the "Small" typeScale size, but there are other properties that I want to apply to all inputLabels for which I don't want to allow change. (Like bottom padding for when they appear above an input etc...)

Comment (villanuevawill) [reply src/components/Text/index.js]: ahhh I see. Ok - lets keep it this way for now. If we find we need to add a few more variations, then we should revisit.

Comment (corwinharrell) [reply src/components/Text/index.js]: 👍

**Pull request merged**